main.go

```diff
@@ -8,6 +8,11 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/rancherio/go-machine-service/events"
 	"github.com/rancherio/go-machine-service/handlers"
+	_ "github.com/rancherio/go-machine-service/handlers/providers/amazonec2"
```

Comment:
if they are not added, those files are never loaded. So, the providers for the clouds are not loaded and saved in the map unless they are loaded. 

The reason we need it here is the same machine keeps it in its main file. I have it load this before loading handlers, because handlers depends on this.